Pipfile

```diff
@@ -16,9 +16,9 @@ yaspin = "*"
 texttable = "*"
 pyyaml = "*"
 requests = "*"
-contoml = "*"
 termcolor = "*"
 invectio = "*"
+toml = "*"
```

Comment:
could you create a micro-test that verifies that this works with nested empty dict?! ;)

Reply:
Done. We should enable zuul's test.